.github/workflows/docker-publish.yaml

```diff
@@ -25,14 +25,13 @@ jobs:
       run:
         working-directory: server/
 
-
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: true
 
       - name: Build image
-        run: docker build . --file dockerfile --tag $IMAGE_NAME
+        run: docker build . --file .dockerfile --tag $IMAGE_NAME
```

Comment:
any specific reason to name it `.dockerfile` ?

Reply:
it will be hidden in `ls` is that what u want ?

Reply:
nothing,I will change the name